test/parallel/test-vm-module-synthetic.js

```diff
@@ -65,4 +65,15 @@ const assert = require('assert');
       code: 'ERR_VM_MODULE_STATUS',
     });
   }
+
+  {
+    assert.throws(() => {
+      Object.getOwnPropertyDescriptor(SyntheticModule.prototype, 'setExport')
+        .value.call('bar', 'baz');
```

Comment:
```suggestion
      SyntheticModule.prototype.setExport.call({}, 'baz');
```

I find it easier to read. Just a nit that can be ignore.

Reply:
Yes, totally, thank you! Fixing it.